### ISSUE TITLE
backend: Attempts to use Configuration table before running the create table statement

### ISSUE THREAD

**Issue by frozenfrank** [softwareconstruction240/autograder#346]
Recent changes have created a new table, but other changes attempt to put default values into the table before the create table statements are run. This causes an issue when the table doesn't already exist.

For an individual, it's pretty straight forward to just run the create table statement, but the system should be able to do it automatically.

See the screenshot below:
<img width="1026" alt="image" src="https://github.com/softwareconstruction240/autograder/assets/19808440/8f1bf197-af45-4951-a0a0-05ed00b07043">

For reference, here is the create table statement:
```sql
CREATE TABLE IF NOT EXISTS `configuration` (
        `config_key` VARCHAR(50) NOT NULL,
        `value` TEXT NOT NULL,
        PRIMARY KEY (`config_key`)
    ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci
```

**Comment by frozenfrank** [comment]
@19mdavenport Can you help with this one?

**Comment by 19mdavenport** [comment]
resolved by #352
